### PR TITLE
Pass WAI-ARIA states into dropdown

### PR DESCRIPTION
**Pull request by jmacqueen**
This will pass the following WAI-ARIA relationship and widget attributes through to `basic-dropdown` in an attempt to complete cibernox/ember-power-select#293

I have a thing about alphabetizing long property lists to help out when exploring or debugging, I hope that doesn't cause too much trouble. Other than alphabetizing, these are the only actual additions to the templates.

`power-select` attribute  | WAI-ARIA attribute in DOM |  Note
--------------------------------|--------------------------------------|----------
`ariaDescribedBy`   | `aria-describedby`
`ariaInvalid `            | `aria-invalid`
`ariaLabel `              | `aria-label`                     | This one still needs to be added to `basic-dropdown`, but inclusion here ahead of time doesn't break anything
`ariaLabelledBy `     | `aria-labelledby`
`required`                 |`aria-required`

I didn't include tests because I'm a bad person. And I didn't want to take the time on the off-chance these changes don't go in :smile:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,7 +67,6 @@
     "ember-cli-babel": "^5.1.5",
     "ember-cli-htmlbars": "^1.0.1",
     "ember-basic-dropdown": "^0.9.5-beta.10",
-    "ember-hash-helper-polyfill": "^0.1.0",
```
Comment (cibernox) [inline package.json]: I agree that I not longer need this polyfill, but it concerns me that [this test](https://github.com/cibernox/ember-power-select/blob/master/tests/integration/components/power-select/customization-with-compoments-test.js#L113) is not failing in 1.13 in travis.

Nothing for your to change, just a note to myself.

**Comment by cibernox** [comment]
I don't mind the alphabetically order. I started grouping properties "conceptually", but clearly there was a moment where that stopped scaling :sweat_smile: . This is better now.

This **does** need tests tho (and probably some work in ember-basic-dropdown itself).

**Comment by cibernox** [comment]
Tests in ember-basic-dropdown and ember-power-select overlap a bit, but I try to have tests in EPS for behaviours that are anyway tested in EBD because sometimes they depend of EPS forwarding the proper properties, so in this case the duplication has a purpose.

**Comment by jmacqueen** [comment]
I've ended up duplicating in similar ways (and for similar reasons) in the widget set I'm making at work. It's helped to catch refactoring problems when I thought I was safely changing something isolated in one component...and it ends up I was wrong!

Especially considering the case of integration tests, in my opinion, it's well worth having all the relevant tests for the EPS interface in the EPS repo...even if it means tests aren't technically as DRY as they could possibly be.

**Comment by jmacqueen** [comment]
Tests added. I went ahead and put in tests for `aria-label` that will fail until `ariaLabel` is added to `cibernox/ember-basic-dropdown` and the updated version included.

**Comment by cibernox** [comment]
@jmacqueen mmm... why travis is not running for the last commit? Also, should we update ember-basic-dropdown first?

**Comment by cibernox** [comment]
Rebase over master to get the version of EPS with support for aria-label.

**Comment by cibernox** [comment]
Thanks for all the a11y work. 

This starts to look complete.

**Pull request merged**